content/conditions/hernia/surgery-guide/section-2.md

```diff
@@ -1,3 +1,7 @@
+### Approx 6 weeks{.label}
```

Comment:
I'm not entirely comfortable using `h3` here before `h2`. Is this really a heading?

And visually it also looks like it has a divider. Feels like the time estimation should follow the subject of the section which is the `h2`.